app/views/content/shared/quotes/_quote_progression_aaron.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= render Content::QuoteComponent.new(
+  text: "The continued professional development I've received throughout my career so far has shaped me.",
+  name: "Aaron, science teacher",
+  large: false
```

Comment:
False is the default for the quote component, so you should be able to remove this line